scripts/travisci/check_tests.sh

```diff
@@ -1,6 +1,76 @@
 #!/usr/bin/env bash
+
```

Comment:
I don't think we should have scripts this complicated in the CI stack. Just have different scripts for the cron job versus the regular builds.

You don't want to report coverage for the cron_job--it will make the coverage system think that the base coverage on master is changing. The coverage system just uses the latest report from master. The cron job doesn't cover much of the code, so I think the coverage rating will start oscillating based on whether the cron or master job ran last.